### DISCUSSION TITLE
Link on avatar

### DISCUSSION THREAD

**Discussion by matthiasbuchner** [CaiJimmy/hugo-theme-stack#373] [Q&A]
Is there a way to set a link on the avatar? On most websites, it is configured to send the user back to the homepage and this is what  I have been trying to do.

**Comment by CaiJimmy** [comment]
Nope. You can edit the theme file to archive this:

https://github.com/CaiJimmy/hugo-theme-stack/blob/57bfea833850e0565cf81d9a26c52359427151a2/layouts/partials/sidebar/left.html#L11-L29
